btree/btree.c

```diff
@@ -3490,6 +3510,7 @@ static struct m0_sm_trans_descr btree_trans[] = {
 	{ "open/create-init", P_INIT, P_ACT  },
 	{ "open/create-act", P_ACT, P_DONE },
 	{ "destroy", P_INIT, P_DONE},
+	{ "close-init", P_INIT, P_INIT},
```

Comment:
PINIT->PINIT transition is not required anymore.

Reply:
Don't you want to add transitions from P_INIT->P_TIMECHECK and vice versa

Reply:
Oh I missed this.
Weirdly enough I didnt get any errors!